### PR TITLE
Check if node has initialized vp before closing mps (to avoid nodeETHMatched issues)

### PR DESCRIPTION
**Pull request by 0xfornax**

### CODE REVIEW

File: rocketpool-cli/minipool/close.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ func closeMinipools(c *cli.Context) error {
 		return nil
 	}
 
+	hasOnly16ETHMps := true
```
Comment (jshufro) [inline rocketpool-cli/minipool/close.go:47]: we should probably print a message and exit before this if `len(details.Details) == 0`

File: rocketpool-cli/minipool/close.go [hunk 2]
```diff
@@ -72,6 +73,14 @@ func closeMinipools(c *cli.Context) error {
 				unwithdrawnMinipools = append(unwithdrawnMinipools, mp)
 			}
 		}
+		if eth.WeiToEth(mp.DepositBalance) != 16 {
```
Comment (jshufro) [inline rocketpool-cli/minipool/close.go:76]: we should move this to L61 so we can refactor the `if mp.CanClose` case to contain a `continue`...

Comment (0xfornax) [reply rocketpool-cli/minipool/close.go:76]: I don't see how this would change an specific minipool CanClose info.

Comment (jshufro) [reply rocketpool-cli/minipool/close.go:76]: it would just make the code more future proof

**Pull request merged**